client/core/core_test.go

```diff
@@ -4806,70 +4816,6 @@ func TestReconfigureWallet(t *testing.T) {
 	}
 }
 
-func TestSetWalletPassword(t *testing.T) {
```

Comment:
Why remove a test? It would be much better to adjust it so that it works with the changes.

Reply:
Hrmmm, `SetWalletPassword` isn't exported any more from core and the logic was moved to `RecofnigureWallet`
so I added one test case to the ReconfigureWallet tests to test settings new password using `ReconfigureWallet`